content/en/metrics/open_telemetry/_index.md

```diff
@@ -155,8 +155,8 @@ The OpenTelemetry Collector has [two primary deployment methods][14]: Agent and
 [2]: https://www.cncf.io/
 [3]: https://www.datadoghq.com/pricing/?product=infrastructure#infrastructure
 [4]: https://opentelemetry.io/docs/instrumentation/
-[5]: /tracing/trace_collection/open_standards/otlp_ingest_in_the_agent/?tab=host#enabling-otlp-ingestion-on-the-datadog-agent
```

Comment:
Should we move the limitations table to the main OpenTelemetry page? It seems a bit out of place here, since it mentions logs and traces